$NetBSD: patch-setup.cfg,v 1.1 2019/09/19 21:25:52 adam Exp $

[pytest] section in setup.cfg files is no longer supported.

--- setup.cfg.orig	2019-09-19 21:12:50.000000000 +0000
+++ setup.cfg
@@ -1,4 +1,4 @@
-[pytest]
+[tool:pytest]
 norecursedirs = .tox _build tor chutney
 
 [bdist_wheel]
